frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepsPicker.tsx

```diff
@@ -33,7 +33,7 @@ export function FunnelStepsPicker(): JSX.Element | null {
                 <Select.Option key={stepIndex} value={stepIndex} label={`Step ${stepIndex + 1}`}>
                     <div style={{ display: 'flex', flexDirection: 'row' }}>
                         Step {stepIndex + 1} (
-                        <EntityFilterInfo filter={stepFilter as EntityFilter} showSubTitle={false} />)
```

Comment:
Do we want to leave this as it? It looks a bit wonky with the 2 sets of parenthesis - don't feel that strongly though
![image](https://user-images.githubusercontent.com/4813045/146457728-a8b61a51-60b2-4857-a87d-36e6beb91abe.png)

Reply:
Hm good find, I think it'd be best if we hide the custom name here. Maybe that prop _will_ come in handy after all!